src/uwtools/rocoto.py

```diff
@@ -25,20 +35,110 @@ def _add_jobname(tree: dict) -> None:
             _add_jobname(subtree)
 
 
-# Public functions
-def write_rocoto_xml(input_yaml: str, input_template: str, rendered_output: str) -> None:
+def _add_tasks(
+    input_yaml: OptionalPath = None,
+) -> None:
     """
-    Main entry point.
+    Define "task" elements in the given config tree and request a "jobname" for each.
```

Comment:
I'm not sure about this docstring:
1. I wouldn't think to call a path to a YAML file a "config tree".
2. I don't see where "task" elements are being defined -- they're just there or not there, right?
Maybe something like
> Load YAML config and add job names to each defined workflow task.

?

Reply:
Yeah, when I moved this out, I did not go back and review the docstring here. Something like that is better, I'll fix it